x-pack/plugins/security_solution/public/flyout/left/components/hooks/use_threat_intelligence_details.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMemo } from 'react';
+import {
+  filterDuplicateEnrichments,
+  getEnrichmentFields,
+  parseExistingEnrichments,
+  timelineDataToEnrichment,
+} from '../../../../common/components/event_details/cti_details/helpers';
+import { SecurityPageName } from '../../../../../common/constants';
+import { SourcererScopeName } from '../../../../common/store/sourcerer/model';
+
+import { useInvestigationTimeEnrichment } from '../../../../common/containers/cti/event_enrichment';
+import { useTimelineEventsDetails } from '../../../../timelines/containers/details';
+import { useSourcererDataView } from '../../../../common/containers/sourcerer';
+import { useRouteSpy } from '../../../../common/utils/route/use_route_spy';
+import { useLeftPanelContext } from '../../context';
+
+export const useThreatIntelligenceDetails = () => {
+  const isAlert = true;
```

Comment:
we could retrieve the `isAlert` value like we do [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/flyout/right/hooks/use_fetch_threat_intelligence.ts#L62)?

Reply:
done, thanks for the snippet. completly forgot that I placed this hack here.